closure/templates/test/BUILD

```diff
@@ -116,6 +128,18 @@ closure_js_test(
     ],
 )
 
+closure_js_binary(
+    name = "greeter_idom_fail_bin",
+    language = "ECMASCRIPT5_STRICT",
+    deps = [
+        ":greeter_idom_fail_lib",
+        "//closure/library",
+        "//closure/library:testing",
+    ],
+    pedantic = True,
+    internal_expect_failure = True,
```

Comment:
I was very happy when I saw it as an option! Outstanding work on the design to make it so simple.

One thing that I noticed was I needed to make the test a `closure_js_binary()` as `closure_js_test()` does not allow `internal_expect_failure`. On one hand this makes sense since the compile will fail so the test will (obviously) never be run. On the other hand it seems like a slightly asymmetrical API.

Can you share your thoughts here?